extensions/wikia/SEOTweaks/SEOTweaksHooksHelper.class.php

```diff
@@ -15,6 +15,159 @@ class SEOTweaksHooksHelper {
 	 */
 	const SHARING_HOSTS_REGEX = '/\.(facebook|twitter|google)\./is';
 
+
+    /**
+     * Size of preferred article image.
+     */
+    const MIN_WIDTH = 200;
+    const MIN_HEIGHT = 200;
+    const MAX_WIDTH = 2000;
+    const PREF_WIDTH = 600;
+    const PREF_HEIGHT = 315;
+
+    /**
+     * Given a file, return url to file, or thumbnail url if size larger than MAX_WIDTH
+     * @param $file File
+     * @return string|false
+     */
+    static protected function getResizeImageUrlIfLargerThanMax( $file ) {
+        $fileUrl = false;
+        $width = $file->getWidth();
+        if ( $width > self::MAX_WIDTH ) {
+            $thumbObj = $file->transform( [ 'width' => self::MAX_WIDTH ], 0 );
+            if ( $thumbObj ) $fileUrl = $thumbObj->getUrl();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add curly braces around the nested statement(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS121)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') 2 statements were found on this line. Reformat the code to have only one statement per line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS122)